src/controllers/midi/midioutputhandler.cpp

```diff
@@ -8,18 +8,20 @@
 #include "moc_midioutputhandler.cpp"
 
 MidiOutputHandler::MidiOutputHandler(MidiController* controller,
-        const MidiOutputMapping& mapping)
+        const MidiOutputMapping& mapping,
+        const RuntimeLoggingCategory& logger)
         : m_pController(controller),
           m_mapping(mapping),
           m_cos(mapping.controlKey, this, ControlFlag::NoAssertIfMissing),
-          m_lastVal(-1) { // -1 = virgin
+          m_lastVal(-1),
+          m_logger(logger) { // -1 = virgin
```

Comment:
Comment on wrong line. Delete it. Should be replaced by a constant eventually.

Reply:
And please don't name the constant "virgin"...

Reply:
Don't worry, this isn't Emacs ruled by a self-appointed dictator for life without any more constructive things to say. I changed it to actually describe what this constant is https://github.com/mixxxdj/mixxx/pull/4239/commits/50c2ab588cbd0eaaa48356117072e3f5b8e5a903